samples/CloudClipboard/LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
```

Comment:
There's already a license for the repository; we shouldn't include another copy.

Reply:
Removed